### PR TITLE
complete both mobile and desktop version

### PR DESCRIPTION
**Pull request by fatmahussein**
-  I have created only the "Home Page" and the "About Page" as the project requirement.
-  I have developed this project using the mobile-first approach. 
- The mobile menu toggles as per the project requirement using javascript.
- The "Featured Speakers" section is created dynamically using javascript.
- The links to the pages have been linked correctly.

Link to the recording : https://www.loom.com/share/6993b65a8d1e4ac3a191a120bc9a07fe?sid=b9478d1d-963a-4f16-8bc0-8b3926f989ee

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+<body>
+    <nav>
+        <div class="upper-nav">
+            <ul class="top">
+                <li class="fb"><img  id="fb" src="assets/fb (1).png" alt="facebook icon"/></li>
+                <li class="twitter"><img id="twitter" src="assets/twitter.png" alt="twitter icon"></li>
+                <li class="pa"><p>English</p></li>
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
```
Comment (wayungi) [inline about.html:18]: - [ ] Please not that the design has not been fully followed in your navigation bar implementation.
**DESIGN**
nav1

**YOUR WORK**
nav2

- [ ] Please move the logo off the edge as in the Desgin 👍🏾 
- [ ] Position the top navigtion items over the News & campaign links 👍🏾

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,653 @@
...
+footer {
+  background-color: #fff;
+  padding: 15px 60px;
+  justify-content: space-around;
+  display: flex;
+  flex-direction: row;
+}
+
+.copyright {
+  text-align: left;
```
Comment (wayungi) [inline styles.css]: - [ ] Please align the logo to the left and text to the right on the desktop size. The is a project requirement. 👍🏾

File: about.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        <li><img class="partner-logo" src="assets/airbnb.png" alt="airbnb"/></li>
+      </ul>
+    </ul>
+  </section>
+  <!-- footer -->
+  <footer class="footer">
+    <div class="footer-logo-container"><img  class="footer-logo" src="assets/Earth.png" alt="logo"/></div>
+    <div>
+    <p class="copyright">2023 Green Earth Global. <br> Some Rights Reserved.</p></div> 
+  </footer>
```
Comment (wayungi) [inline about.html]: - [ ] Please not the difference between the about page footer as highlighted below and implement these changes on your project in order to meet the design specifications

**Design**
footer1

**Your implementation**
footer

- [ ] Please delete the partners section 👍🏾 
- [ ] Give the footer a black color 👍🏾

File: index.html [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    </navbar>
+   <!--Header-->
+   <header>
+    <h1 class="heading1">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
+    <p  class="parag">Welcome to our global annual summit that will be held virtually for ease of accessibility at your comfort.
+        For those who want to attend in person, it will be held at Uhuru Park, Nairobi,Kenya. </p>
+    <!-- Today's date -->
+    <h3 class="date"></h3>
+   </header>
```
Comment (wayungi) [inline index.html]: - [ ] Kindly not the missing part labeled **5** in your implementation below.

**Design**
top1

**Your implementation**
top2

- [ ] Please kindly stick to the font size & colors. Your are supposed to only change images & texts to customize the page but retain the original design by Cindy  👍🏾 

- [ ] The pages should use the exact same colors, for text, backgrounds, borders, etc, from the original design.

**Basic color swatch: #272A31, #EC5242, #D3D3D3**

File: index.html [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+            </div>
+        </div>
+        <div class="grid-item2">
+            <h3 class="join-p">Join Green Earth Global Summit</h3>
+        </div>
+
+    </div>
+    <h3 class="heading31">SEE THE WHOLE PROGRAM</h3>
+    <hr class="hrprog">
+   </section>
```
Comment (wayungi) [inline index.html:109]: - [ ] Kindly not the areas where your implementation has diverged from the design as pointed out below. Please fix the changes. 

**DESIGN**
midl1

**YOUR IMPLEMENTATION**
midl2

- [ ] For each article 
   -  Align the content in the center (title, image & description)
   -  Give each article a transparent background color
   
- [ ] The main section has a background pattern, please apply that too. 

**REMEMBER THIS**
- [ ] The pages should look almost identical to the original design, outside the content differences.
- [ ]The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

**Basic color swatch: #272A31, #EC5242, #D3D3D3**

File: index.html [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+
+   <!-- Featured speakers -->
+   <section class="Featured">
+  <h4 class="speakers">Featured Speakers</h4>
```
Comment (wayungi) [inline index.html:113]: - [ ] Kindly add some spacing between the speakers title and the section above it as implemented in the design

**DESIGN**
SPK1

**Your implementation**
SPK2

File: index.html [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/recycle.png" alt="Recycle"/>
+                <p class="first-p">Recycle</p>
+                <p class="second-p">Plastic waste contribute to a huge percent of sea animals suffocation.</p>
+            </div>
+        </div>
+        <div class="grid-item2">
+            <h3 class="join-p">Join Green Earth Global Summit</h3>
```
Comment (wayungi) [inline index.html:103]: - [ ] For mobile screens, align this text in center as in the design please. 
btn

File: about.html [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+              </li>    
+          </ul>
+          </div>
+    </nav>
+   <!--Header-->
+   <header>
+    <h1 class="heading1about">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
+    <p  class="paragabout">Human existence is quite impossible without the presence of a healthy ecosystem. Our environment comprises all living and non-living components and their interactions within a natural habitat. Environmental conservation has become one of the core issues that need to be addressed to battle climate change and global warming.Green Earth Global aims at conservation of natural resources. </p>
+      </header>
```
Comment (wayungi) [inline about.html]: - [ ] Please not the design discrepancies in your implementation as highlighted below. I kindly suggest you fix them all. 😅 

**Design**
top1

**Implementation**
top2

- [ ] Note that the label **4** does not exist on the about page. You seem to have duplicated the home page here ⚠️ 
- [ ]  You have the contact information in a brow belt but that does not exist in the original design about page. kindly fix ⛑️ 

**PLEASE NOTE**
- [ ] The pages should look almost identical to the original design, outside the content differences.
- [ ] The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

* *Basic color swatch: #272A31, #EC5242, #D3D3D3**

- [ ] The pages use the same fonts or simialr fonts from the original design.

File: about.html [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+<section class="past-summit">
+    <h3 class="summit">Green Earth Global Summit 2020</h3>
+    <hr class="hrabout">
+    <p class="majisafiP">The logo of Green Earth organisation was changed as of 21st July, 2020. We thank everyone who took part in voting for the logo below.</p>
+    <img class="majisafi" src="assets/Earth.png" alt="logo"/>
+</section>
+
+<!-- Past pictures -->
+<section class="pastphotos">
+    <h4 class="pastp">See the past G.E Global summit </h4>
```
Comment (wayungi) [inline about.html]: - [ ] Please note the following design discrepancies and align them in your implementation

**Design**
midl1

**Impelentation**
midl2

**Please take the following action**
- [ ] Add spacing between the title & the top section 😀 
- [ ] Color the divers red 🦆 
- [ ] Note the dividing line and add it 👍🏾

File: about.html [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+    <p class="pag"> Take a look at the past two G.E global summit which took place<br>in Buenos Aires and Warsaw.</p>
+    <div >
+        <ul class="group-photo">
+        <li >
+             <img class="grp" src="assets/group1.png" alt="group photo"/>
+                </li>
+        <li ><img class="grp" src="assets/group2.png" alt="group photo"/>
+            </li>
+    </ul>
+    </div>
```
Comment (wayungi) [inline about.html:86]: - [ ] Your past event images are too small on the mobile versions. Please just change their alignment but keep their sizes almost the same.

**Design**
img1

**Your project**
img2

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @fatmahussein , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks passing ✔️ 
- [ ] Descriptive PR body ✔️ 
- [ ] Good attempt at following the design ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please deploy your project using github page. The link is broken currently. If yu have deployed the project, please update the link.
demo

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+    <nav>
+        <div class="upper-nav">
+            <ul class="top">
+                <li class="fb"><img  id="fb" src="assets/fb (1).png" alt="facebook icon"/></li>
+                <li class="twitter"><img id="twitter" src="assets/twitter.png" alt="twitter icon"></li>
+                <li class="pa"><p>English</p></li>
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
+        </div>
```
Comment (Ofarouq310) [inline about.html:19]: - [ ] Kindly note that your navigation bars have to be aligned together as in the same design, kindly fix that.

| Your design | Actual design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+        </div>
+
+    </div>
+    <h3 class="heading31">SEE THE WHOLE PROGRAM</h3>
+    <hr class="hrprog">
+   </section>
```
Comment (Ofarouq310) [inline index.html:109]: - [ ] Kindly note that you have to stick to the design guidelines, kindly address the following:

   -  Add the pattern background included in the design.
   -  Make sure to add the hover effect on the cards as shown in the design.
   -  Make sure to use the basic color swatch: #272A31, #EC5242, #D3D3D3
  
| Your design | Actual design |
| ------------- | ------------- |
| image  | image |

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @fatmahussein, ✋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🥈

giphy

 ## Highlights
✅ Descriptive PR title & summary.

 ## Required Changes ♻️

- [ ] Kindly note that you have to add a deployment link for your project inside your README file, inside the `LIve Demo` section.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**